app/static/css/main.css

```diff
@@ -0,0 +1,34 @@
+body {
```

Comment:
Because this is just output from our compiled sass file, it would be great if we didn't track it in our git history. Could you this file to our .gitignore?